applications/nrf5340_audio/src/bluetooth/ble_hci_vsc.h

```diff
@@ -56,6 +66,12 @@ struct ble_hci_vs_cp_set_radio_fe_cfg {
 	uint8_t ant_id;
 } __packed;
 
+enum ble_hci_vs_max_tx_power {
```

Comment:
should "fem" be part of this name?

Reply:
No, since you can set it to +3dBm without the FEM